util/dvsim/sim_utils.py

```diff
@@ -99,3 +99,76 @@ def vcs_cov_summary_table(buf):
 
     # If we reached here, then we were unable to extract the coverage.
     raise SyntaxError(f"Coverage data not found in {buf.name}!")
+
+
+# Capture and return two runtimes: job_runtime and simulated_time.
+# The run.log file is passed as an input to the function, in addition to the
+# tool used. The tool returns two integers if the runtimes were extracted
+# successfully. The function will raise appropriate exception if the extraction
+# fails.
+def get_runtimes(run_log_txt, tool):
+    with open(run_log_txt, 'r') as f:
+        if tool == 'xcelium':
+            return xcelium_runtimes(f)
+        elif tool == 'vcs':
+            return vcs_runtimes(f)
+        else:
+            # TODO: support using default dvsim run time
+            raise NotImplementedError(f"{tool} is unsupported for run times extraction.")
+
+
+def vcs_runtimes(buf):
+    job_runtime = 0
```

Comment:
Please add docstrings for all of these utility methods.

Reply:
Please indicate a real example of the line containing the timing info printed by the simulator.